packages/frontend/component/src/components/auth-components/onboarding-page.tsx

```diff
@@ -37,7 +37,7 @@ function getCallbackUrl(location: Location) {
   try {
     const url =
       location.state?.callbackURL ||
-      new URLSearchParams(location.search).get('callbackUrl');
+      new URLSearchParams(location.search).get('redirect_uri');
```

Comment:
Will it be incompatible?

Reply:
nope, the onboarding page never get callback url